content/app-dev/create-actor/run.md

```diff
@@ -1,50 +1,70 @@
 ---
-title: "Run the Actor"
-date: 2018-12-29T11:02:05+06:00
-weight: 5
+title: "Running the actor"
+date: 2018-12-29T10:00:00+00:00
+weight: 4
 draft: false
 ---
 
-In this guide we're going to start the actor "the long way" so that you can get a feel for all of the moving parts of the process. Our tooling documentation should help you get actors started easier once you've been through this guide.
+In [Getting started](/overview/getting-started) you ran an actor from the dashboard UI. Now, we're going to start the actor "the long way" so that you can get a feel for all of the moving parts of the process. Our tooling documentation should help you get actors started more easily, once you've been through this guide.
 
-### Pre-Requisites
-To run this part of the guide, you'll need `wash`. If you don't have `wash`, installation steps are covered on [the installation page](../../../overview/installation)).
+We assume you've already [installed](../../../overview/installation)) wash, the wasmCloud host, and necessary prerequisites.
+
+### Build the actor
+
+Building the actor is as easy as running `make` in the project's root directory. This will compile the actor and sign it with a freshly generated seed key. The `Makefile` that comes with the newly generated project lists claims for the actor including `wasmcloud:httpserver`, so it can work with the HTTP Server capability provider.
 
 ### Launch the Actor
 
-There are countless ways to run the actor we just created, but the easiest is probably to use our **REPL**, by typing `wash up` at the command line. You should see a screen that looks similar to the following:
+There are countless ways to run the actor we just created, and if you went through the [getting started](../../../overview/getting-started) section, you'll have seen some some of them already. Open your browser to view the web dashboard  [http://localhost:4000](http://localhost:4000) to view the dashboard (you may need to modify the port number`4000` if you changed it before launching the wasmCloud host).
+
+At this point, simply click **Start Actor** and then choose _from file_. Open a browser to your current project directory and then navigate to the `build` directory. Here you'll find a `(project)_s.wasm` file, where `(project)` is `hello`, or the name you selected for the project. The `_s` suffix indicates that this WebAssembly module has been _signed_.
+
+A moment later you should see the actor in your web UI as shown in the following screenshot:
+
+TODO screenshot actor started
```

Comment:
When I add the screenshot here, I'll also add a snippet telling the user to select the local-host from the host selector. If it 's not selected then the `from file` option won't display

Reply:
NVM, fixing in https://github.com/wasmCloud/wasmcloud-otp/issues/221